client/components/content-containers/tcc-exam/info.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
-  type: 'EXAM',
+  templateId: 'EXAM',
```

Comment:
No but we can. The reason I did not do it because AICPA was the only project at the time that was using it plus this is not an officially released feature yet.